### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by lajarre**
A lot of stuff going on here, as explained by the commit messages.

While adding the CONTRIBUTING docs, I felt the urge to:

- rename `scheduler` to `backend`
- add linting checks to CI.

@acemasterjb you should `pip install -r dev-requirements.txt` from now on, and install all linters (as mentioned in the CONTRIBUTING file) to your editor.

For now, pylint is disabled in CI because it's raising too many problems, but let's add it to our editors and fix files little-by-little from now on.

EDIT: finally I've split the code changes in several different PRs:

- reformat frontend code: https://github.com/butterdao/whip/pull/33
- reformat backend code: https://github.com/butterdao/whip/pull/35
- add linting to ci: https://github.com/butterdao/whip/pull/34

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,4 +10,53 @@ Quantify Treasury Risk exposure with Whip's Risk Analytics Dashbord. Identify ri
...
+
+Env vars to define:
+
+
+- frontend:
+  - `API_URL`
+- backend: the same as in `backend/app.env`.
+
+For now the backend is playing with CORS, so you want to play with a proxy for HTTP calls from the
+frotnend if deploying frontend and backend to different (sub-)domains.
```
Comment (vaughnmck) [inline README.md:57]: small typo on 'frontend'

**Review by vaughnmck — CHANGES_REQUESTED** [review]
Small typo on line 57 of the README but otherwise happy to merge.

Can you also update the contributing link on line 3 of the README?

File: README.md [hunk 2]
```diff
@@ -10,4 +10,53 @@ Quantify Treasury Risk exposure with Whip's Risk Analytics Dashbord. Identify ri
...
+
+```sh
+docker compose up
+````
+
+If you need to install Python locally (for development), install with:
+
+```sh
+pip install -r dev-requirements.txt
+```
```
Comment (acemasterjb) [inline README.md:45]: Has the `dev-requirements.txt` file been pushed yet?

Took a look in the dirs but only found `requirements.txt`

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+What we like:
+
+- Small PRs.
+- Commit messages that are [clean](https://cbea.ms/git-commit/) but also [fun](https://gitmoji.dev/).
+- Writing unit tests for critical code.
+
+What we use (you should install these tools in your favorite editor):
+
+- Javascript: typescript, eslint, prettier.
+- Python: pylint, black, isort, flake8, mypy.
```
Comment (acemasterjb) [inline CONTRIBUTING.md:18]: Maybe we should mention the test suite of choice (pytest) somewhere in here as well?

**Comment by acemasterjb** [comment]
Also, is `scheduler` meant to be renamed to `backend` in a future PR? If so, I agree that it should be renamed to `backend` given its contents.

File: README.md [hunk 4]
```diff
@@ -10,4 +10,53 @@ Quantify Treasury Risk exposure with Whip's Risk Analytics Dashbord. Identify ri
...
+then open `backend/app.env` and edit the environment variables with the right API keys and such.
+
+
+For development work, you probably want to run the frontend dev server in your own shell:
+
+```sh
+# Shell 1:
+docker compose --profile backend up
+# Shell 2:
+cd frontend && npm run dev
```
Comment (acemasterjb) [inline README.md:32]: maybe we should mention that the user should install frontend deps with `npm i` before running npm scripts

**Review by acemasterjb — CHANGES_REQUESTED** [review]

**Comment by lajarre** [comment]
After having addressed all pending comments, I'll take the liberty to merge to move forward as this is non critical. Please add more comments if needed, I'll address them separately.

**Pull request merged**